### ISSUE TITLE
Archiving a favorites page breaks the menu and it is no longer usable

### ISSUE THREAD

**Issue by josemejias11** [dotCMS/core#24603]
### Parent Issue
#22343 

### Problem Statement

Trying to archive a favorites page breaks the Menu and it becomes unclickable 

### Steps to Reproduce

1. Go to https://localhost:8443/c
2. Add a page to favorites
3. Try to archive the page
4. Click on any element of the menu

### Acceptance Criteria

Menu should still be available after archiving any page

### dotCMS Version

23.05

### Proposed Objective

Core Features

### Proposed Priority

Priority 2 - Important

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

https://user-images.githubusercontent.com/129882031/231295362-5b3bccc4-f1f3-4bdb-9f51-d9ab6aacddaa.mov

_No response_

**Comment by fmontes** [comment]
@bryanboza @josemejias11 

Please try this in master. It's not happening to me. 

When it was tested that time, the archive was done to the content type that saves the favorites and not to the page. Now, if I try to do it on the page.

I see another issue: when I archive and the favorite remains, if I try to edit it, it throws a 404 error. So I will be creating that one.

**Comment by josemejias11** [comment]
Blocked by https://github.com/dotCMS/core/issues/24861

**Comment by josemejias11** [comment]
**Approved QA** - Tested on **23.06_21c1c10b_SNAPSHOT** // Docker // macOS 13.0 // FF v113.0
